src/styles/App.css

```diff
@@ -20,3 +20,29 @@ a {
   color: var(--text);
   text-decoration: none;
 }
+
+body {
+  background: var(--background);
+  color: var(--text);
+  /* display: flex; */
+  justify-content: space-around;
+  font-size: 17px;
+  line-height: 122%;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.plus {
```

Comment:
this seems specific to the plus component so it should live in the component file